index.js

```diff
@@ -34,6 +34,7 @@ export const VERSION_1_MODEL_B_REV_2 = 'rpi1_b2';
 export const VERSION_1_MODEL_B_PLUS = 'rpi1_bplus';
 export const VERSION_1_MODEL_A_PLUS = 'rpi1_aplus';
 export const VERSION_2_MODEL_B = 'rpi2_b';
+export const VERSION_3 = 'rpi3';
```

Comment:
can we change the value to `rpi3_b`, since it's full name is technically the "Raspberry Pi 3 Model B" and also rename the constant to `VERSION_3_MODEL_B`?